kornia/morphology/morphology.py

```diff
@@ -563,3 +563,55 @@ def bottom_hat(
         )
         - tensor
     )
+
+
+def skeletonize(tensor: torch.Tensor, kernel: Optional[torch.Tensor] = None) -> torch.Tensor:
```

Comment:
let it not be a word used so much by torch
```suggestion
def skeletonize(data: torch.Tensor, kernel: Optional[torch.Tensor] = None) -> torch.Tensor:
```